moonworm/cli.py

```diff
@@ -3,6 +3,7 @@
 import os
 from multiprocessing.sharedctypes import Value
 from pathlib import Path
+from posixpath import relpath
```

Comment:
I would prefer `os.path.relpath` here, or `pathlib`.

Reply:
`os.path.relpath`: https://docs.python.org/3/library/os.path.html#os.path.relpath
`pathlib`: `relative_to`: https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.relative_to

Reply:
They are higher level libraries that choose posix path or other modules depend on os, filesystem, etc.